draft-ietf-quic-transport.md

```diff
@@ -1297,10 +1300,10 @@ When implementing the server role, applications need to be able to:
 
 - listen for incoming connections, which prepares for the exchange described in
   {{handshake}};
-- if Early Data is supported, embed application-controlled data in the TLS
+- if 0-RTT data is supported, embed application-controlled data in the TLS
   resumption ticket sent to the client; and
-- if Early Data is supported, retrieve application-controlled data from the
-  client's resumption ticket and enable rejecting Early Data based on that
+- if 0-RTT data is supported, retrieve application-controlled data from the
+  client's resumption ticket and enable rejecting 0-RTT data based on that
   information.
```

Comment:
I think that we need to get @MikeBishop to fight this out.

Reply:
I'm fine with Early Data but it was undefined, so this seemed like the easiest fix. But a defn. would work too.

Reply:
It doesn't help that RFC 8446 uses "0-RTT Data" and "Early Data" almost equal numbers of times.  We should have picked one and stuck to it.  Ah well -- we can do the same thing RFC 8446 does and put ("Early Data") as a synonym after the first mention of 0-RTT.

Reply:
> It doesn't help that RFC 8446 uses "0-RTT Data" and "Early Data" almost equal numbers of times.

Happy to be of service

> Ah well -- we can do the same thing RFC 8446 does and put ("Early Data") as a synonym after the first mention of 0-RTT.

That would be fine. I'm really fine with any resolution here that is clear.